packages/web_benchmarks/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: web_benchmarks
 description: A benchmark harness for performance-testing Flutter apps in Chrome.
 repository: https://github.com/flutter/packages/tree/main/packages/web_benchmarks
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+web_benchmarks%22
-version: 3.0.0
+version: 3.1.0-wip
```

Comment:
Is `wip` a label that's understood by `pub`? Should we use something like `dev` instead, which I believe has special meaning to `pub`. Also, for a small package like this, maybe we don't need a label? Just make it 3.1.0, populate the changelog, and immediately after releasing 3.1.0 we bump it to 3.2.0. WDYT?

Reply:
wip is to indicate the package is still a work in progress and is not ready to publish. If the pubspec version is a clean version like '3.1.0' release github action will run and automatically publish the package.

Reply:
Ah, I wasn't aware of how our github actions worked. Cool!